wes_client/client.py

```diff
@@ -0,0 +1,121 @@
+from logging import Logger
+import socket
+
+import requests
+from requests import Session
+import urllib3
+
+class wesClient:
+    def __init__(self,url:str, base_path:str, token: Optional[str] = None,)-> None:
```

Comment:
Please use a linter to check your code. At least `flake8`, but probably both `flake8` and `pylint`. Good to get used to good practices, makes it easier to read and maintain code.

Also, please use `mypy` to check typing.

Finally, please add [Google-style docstrings](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html) to your class definition and all methods. See TRS-cli and most other more recent repos for examples. When documenting class definitions, note the difference between `Args` (here: `url`, `base_path` and `token`) and `Attributes` (here: `url`, `base_path`, `token`, `headers`, `session`).